packages/template-ui/src/views/Home.vue

```diff
@@ -2,7 +2,7 @@
   <div
     :style="{ backgroundImage: backgroundImageURL }"
   >
-    <ChannelNavBar />
+    <ChannelNavBar :atHome="true" />
```

Comment:
Maybe the `atHome` could be replaced with a check for the node prop. It's not the same? I mean, in this case, `atHome` is used when there's no node to pass to the component, so maybe we can replace that property with a computed one.

Reply:
After going back to the code I realize why the `atHome` option is needed: the Search view doesn't have a node to pass to the header component, but the header should have the link to go back home. Only the Home view doesn't need the link. So I'm leaving this as-is.